cmd/params/params.go

```diff
@@ -534,6 +544,11 @@ func loadProjectParams(v *viper.Viper) (ProjectParams, error) {
 	projectMap := vipertools.GetStringMapString(v, "projectmap")
 
 	for k, s := range projectMap {
+		// make all regex case insensitive
+		if !strings.HasPrefix(k, "(?i)") {
+			k = "(?i)" + k
+		}
+
 		compiled, err := regexp.Compile(k)
```

Comment:
Let's change this to:

```go
compiled, err := regex.Compile(k)
```

So we support negative lookahead for all user input regex patterns.

Reply:
Native lib doesn't support negative/positive lookahead. So if any user has already been using, this version will break for them.

Reply:
That's fine, the project map can always be fixed on the dashboard with custom rules. The ones that can't be fixed after sending are `include` and `exclude`.

Reply:
It's worth it to have all user-facing regex behave the same way.

Reply:
I'll do it in a separate PR